code/modules/mob/living/carbon/human/human_movement.dm

```diff
@@ -79,3 +79,12 @@
 	if(dna.species.space_move(src))
 		return TRUE
 	return ..()
+
+/mob/living/carbon/human/carefulmovement()
```

Comment:
Some of these checks are redundant with the parent and this proc should probably just extend that

Reply:
parent also checks move intent while this needs to check species to check exact speed. this aint feasible